src/Libraries/Microsoft.Extensions.Telemetry/Logging/ExtendedLogger.LegacyTagJoiner.cs

```diff
@@ -76,5 +76,7 @@ public void SetIncomingTags(IReadOnlyList<KeyValuePair<string, object?>> value)
         }
 
         IEnumerator IEnumerable.GetEnumerator() => GetEnumerator();
+
+        public override string? ToString() => State?.ToString();
```

Comment:
As I understand, in case of the modern logging path (State is a LoggerMessageState) we'll output the LoggerMessageState serialized to a string as the Message, right? Is it the desired behavior?